thorlcr/activities/thactivityutil.cpp

```diff
@@ -783,9 +783,26 @@ IFileIO *createMultipleWrite(CActivityBase *activity, IPartDescriptor &partDesc,
     Owned<IFileIO> fileio;
     if (compress)
     {
+        unsigned __int64 compMethod = COMPRESS_METHOD_LZW;
+        // rowdif used if recordSize > 0, else fallback to compMethod
+        if (twFlags & TW_Temporary)
```

Comment:
would be better to pass in the ICompressor (or NULL if not compressing) and therefore not require flag/code here.